docs/design/chart_format.md

```diff
@@ -189,25 +176,14 @@ keywords:
 * proxy
 source: https://github.com/foo/bar
 home: http://nginx.com
-depends:
-        charts:
-          - name: memcached
-                location: helm.sh/charts/memcached
-                version: 0.9.3
-        kubernetes:
-                version: “>=1.0.0”
-                extensions:
-                  - extensions/v1beta1
+expander:
+  name: GoTemplates
```

Comment:
+1 to having a semantics for "no expander".  There is probably a good pattern here -- charts with no expander can live at the root and be closed functions.  I will have to change the doc though to reflect that design.

FWIW We want to merge the charts and modify them so they work and are really useful.  E.g. we will probably want to parameterise most of them, and will have to convert them all to return DM types.  So the expander field isn't a big deal in the scheme of things.